pkg/operator/ceph/cluster/mgr/dashboard.go

```diff
@@ -201,26 +203,35 @@ func (c *Cluster) setLoginCredentials(password string) error {
 	// Set the login credentials. Write the command/args to the debug log so we don't write the password by default to the log.
 	logger.Infof("setting ceph dashboard %q login creds", dashboardUsername)
 
+	// Generate a temp file
+	file, err := ioutil.TempFile("", "")
```

Comment:
Use `defer` to remove the temp file once we exit the function.

Reply:
Done.